### PR TITLE
Update developer docs Gas page

### PR DESCRIPTION
**Pull request by vunderkind**
Updated the gas page:
- Added 'Gasnow' to related tools
- Expanded on the definition of gwei
- Linked article about Proof of Stake as it may be relevant to the future of lower-cost gas on the Ethereum network.

### CODE REVIEW

**Comment by wackerow** [comment]
Hey @vunderkind, thanks for the PR! There is an existing PR (#3104) that also changing this section. It is in final review but we should wait until that is merged before this gets reviewed. Bear with us =)

**Comment by vunderkind** [comment]
> Hey @vunderkind, thanks for the PR! There is an existing PR (#3104) that also changing this section. It is in final review but we should wait until that is merged before this gets reviewed. Bear with us =)

Thanks @wackerow for the update! Fingers crossed.

File: src/content/developers/docs/gas/index.md
Comment (wackerow) [inline src/content/developers/docs/gas/index.md:65]: Given the new changes that have been added to this section regarding London with an expanded explanation, I feel this section may be best left as is. 

Perhaps we could cherry-pick the line about `The word 'gwei' itself means 'giga-wei', and it is equal to 1,000,000,000 wei.` and place below this sentence (which is located higher up on the page in the "What is gas?" section):

> In essence, gas fees are paid in Ethereum's native currency, ether (ETH). Gas prices are denoted in gwei, which itself is a denomination of ETH - each gwei is equal to 0.000000001 ETH (10<sup>-9</sup> ETH). For example, instead of saying that your gas costs 0.000000001 ether, you can say your gas costs 1 gwei. The word 'gwei' itself means 'giga-wei', and it is equal to 1,000,000,000 wei.

Note nine (9) zeros on that sentence as well instead of 6 in your original copy

File: src/content/developers/docs/gas/index.md
Comment (wackerow) [inline src/content/developers/docs/gas/index.md:45]: Consider adjustment:
> Gas also exists to incentivize block producers in the Ethereum network to include transactions. Without this financial incentive, miners could be incentivized to mine empty or incomplete blocks, which could result in denial-of-service and/or censorship.

File: src/content/developers/docs/gas/index.md
Comment (wackerow) [inline src/content/developers/docs/gas/index.md:110]: Reviews about this website have been less than encouraging.
https://chrome.google.com/webstore/detail/gasnow-eth-gasprice-forec/ccnipbjnnkophfgnkmhdongajmkhijbl?hl=en-US
Their browser extension is heavily advertised, and users report that they have to give the extension access to the contents of their clipboard which is an incredible red-flag in regards to security and risk of users having funds stolen.
Would strongly vote against linking to any site with this reputation.

**Review by wackerow — CHANGES_REQUESTED** [review]
@vunderkind Left some comments for ya =)

@minimalsm Feel free to drop your feedback as well on this one given we're trying to finalize this page in proximity to the London upgrade.

**Comment by wackerow** [comment]
This page is again in the midst of more changes to get it prepared for the London upgrade with EIP-1559. @minimalsm You've put a lot into the updates being pushed, curious your thoughts on any of the content suggested here. TBH, much of this seems to now be covered.

**Review by minimalsm — COMMENTED** [review]
> This page is again in the midst of more changes to get it prepared for the London upgrade with EIP-1559. @minimalsm You've put a lot into the updates being pushed, curious your thoughts on any of the content suggested here. TBH, much of this seems to now be covered.

Added some comments. In summary:

- I think the trivia is a nice add but need to fact check origin of wei
- GasNow shouldn't be included due to security concerns
- I don't really see PoS as strongly tied to gas price

**Review by minimalsm — APPROVED** [review]

**Review by minimalsm — APPROVED** [review]

**Pull request merged**
